src/video_core/texture_cache/texture_cache.h

```diff
@@ -776,6 +787,44 @@ class TextureCache {
         }
     }
 
+    /**
+     * `GetNullSurface` gets a null surface based on a target texture.
```

Comment:
```suggestion
     * Gets a null surface based on a target texture.
```

This way you don't need to remember to change the documentation comment if the function name changes.